src/main/java/frc/utils/applicationsutils/CMDHandler.java

```diff
@@ -0,0 +1,24 @@
+package frc.utils.applicationsutils;
+
+import java.io.IOException;
+import java.nio.file.Path;
+
+public class CMDHandler {
```

Comment:
consider adding runPythonFile() and runJavaFile() that gets only file name or path, or maybe give the class as param and use getPackageName() (only for java)